packages/babel-plugin-jest-hoist/src/index.ts

```diff
@@ -73,14 +73,15 @@ Object.keys(global).forEach(name => (WHITELISTED_IDENTIFIERS[name] = true));
 
 const JEST_GLOBAL = {name: 'jest'};
 const IDVisitor = {
-  ReferencedIdentifier(path) {
+  ReferencedIdentifier(path: any) {
     this.ids.add(path);
   },
   blacklist: ['TypeAnnotation', 'TSTypeAnnotation', 'TSTypeReference'],
+  ids: new Set(),
```

Comment:
?

Reply:
It's return `... the ids properties does not exist on this type ...` on line 77 any idea how to fix it?

Reply:
I tried adding the correct type info here, but was unable to. so I just added a `@ts-ignore` with an explanation 🙂